### PR TITLE
[docs] Hide ad for paid docs pages

### PR DESCRIPTION
**Pull request by oliviertassinari**
Something I noticed in #7213. Open, https://next.mui.com/x/react-data-grid/column-ordering/, developers shouldn't see an ad:

Screenshot 2022-12-25 at 13 10 26

### CODE REVIEW

File: docs/data/data-grid/column-ordering/column-ordering.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ By default, columns are ordered according to the order they are included in the
 
 By default, `DataGridPro` allows all column reordering by dragging the header cells and moving them left or right.
 
-{{"demo": "ColumnOrderingGrid.js", "disableAd": true, "bg": "inline"}}
+{{"demo": "ColumnOrderingGrid.js", "bg": "inline"}}
```
Comment (oliviertassinari) [inline docs/data/data-grid/column-ordering/column-ordering.md:13]: The ads on the demos are automatically hidden when the page has the `disableAd` prop. I'm removing the duplication.

File: docs/data/pages.ts [hunk 2]
```diff
@@ -80,8 +80,8 @@ const pages: MuiPage[] = [
           '/x/react-data-grid/pivoting',
         ],
         children: [
-          { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
           { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
+          { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
           { pathname: '/x/react-data-grid/aggregation', title: 'Aggregation', plan: 'premium' },
```
Comment (oliviertassinari) [inline docs/data/pages.ts:85]: I have sorted the side nav with the first community, second pro, and third premium. The idea is that:

1. docs usually display the basic features first, so we communicate that Premium is more advanced than Pro and Pro more advanced than Community.
2. It separates a bit more MIT from the Commercial features, giving more of the sense that MIT can be used standalone.

File: docs/data/pages.ts [hunk 3]
```diff
@@ -80,8 +80,8 @@ const pages: MuiPage[] = [
...
           { pathname: '/x/react-data-grid/pivoting', title: 'Pivoting 🚧', plan: 'premium' },
```
Comment (oliviertassinari) [inline docs/data/pages.ts:86]: I have sorted the side nav with the first community, second pro, and third premium. The idea is that:

1. docs usually display the basic features first, so we can communicate that Premium is more advanced than Pro and Pro more advanced than Community.
2. It separates a bit more MIT from the Commercial features, giving more of the sense that MIT can be used standalone.

File: docs/pages/x/react-date-pickers/legacy-date-picker.js [hunk 4]
```diff
@@ -3,5 +3,5 @@ import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
 import * as pageProps from 'docsx/data/date-pickers/legacy-date-picker/legacy-date-picker.md?@mui/markdown';
 
 export default function Page() {
-  return <MarkdownDocs {...pageProps} />;
+  return <MarkdownDocs {...pageProps} disableAd />;
```
Comment (LukasTy) [inline docs/pages/x/react-date-pickers/legacy-date-picker.js:6]: You probably wanted to update `legacy-date-range-picker` file instead? 🤔

**Pull request merged**